accepted/0000-template-republish-npm-package-after-unpublish.md

```diff
@@ -0,0 +1,89 @@
+# RFC: npm publish, unpublish, and deprecate functionality and data/repository management and log management policy (security policy) changes
+
+## Summary
+
+When I publish a package v1.0.0 and unpublish it, it is unpublished correctly. However, I am not able to re-publish any other codebase B/ C/ D into v1.0.0. I will not be able to re-publish the same version v1.0.0 with a new or same codebase.
```

Comment:
This is good and intended. While it may suck as a publisher (I recently had to help internally with an ancient mispublish that bumped a semver minor, meaning we had to go straight to 4.8.0 rather than starting at 4.0.0!), the existing approach is very rarely an issue, is a simpler solution, and is less potentially harmful than what's proposed here IMO.